lib/hmmlearn/base.py

```diff
@@ -2,11 +2,13 @@
 import string
 import sys
 from collections import deque
+import threading
 
 import numpy as np
 from scipy import linalg, special
 from sklearn.base import BaseEstimator
 from sklearn.utils import check_array, check_random_state
+from joblib import Parallel, delayed
```

Comment:
Can you use stdlib's concurrent.futures instead, which provides AFAICT a very similar API?